### DISCUSSION TITLE
The whisper.store file

### DISCUSSION THREAD

**Discussion by jimhh** [AsamK/signal-cli#1121] [Q&A] [answered]
Hi,

This may be a stupid question due to my lack of general knowledge of how Singnal/Server works, anyway...

I notice there is a KeyStore file `whisper.store` that is imported, and I was curious how this file is created.
There does not seem to be any mention of this file anywhere. If you have the information I would very much like to understand how to create this file.

Any help is much appreciated

**Comment by AsamK** [comment, marked as answer]
It's copied from https://github.com/signalapp/Signal-Android/blob/main/app/src/main/res/raw/whisper.store

**Reply by jimhh** [reply]
Thank you.
